src/github.rs

```diff
@@ -146,8 +147,65 @@ impl GitHubApi {
         }
         Ok(result)
     }
+
+    /// Get all teams for the rust-lang org
+    pub(crate) fn teams(&self) -> Result<Vec<GitHubTeam>, Error> {
+        Ok(self
+            .prepare(true, Method::GET, "orgs/rust-lang/teams?per_page=100")?
```

Comment:
Hm, it seems likely that we should validate we're not hitting pagination limits here. We currently have 119 rust-lang/team teams (not all of which correspond to an entity on GitHub), so it's likely that we'll pretty soon run up against a 100 team limit here if we're not already.

I don't think it's necessarily true that we need to block on doing pagination, but we shouldn't silently skip teams.